spec/factories/user.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :user do
+    email { "user#{rand(1...9999)}@mail.com" }
```

Comment:
Is there a specific reason why we make the email like this?

Reply:
cause we need to have random email